www-client/qutebrowser/qutebrowser-2.5.3.ebuild

```diff
@@ -21,7 +21,7 @@ HOMEPAGE="https://www.qutebrowser.org/"
 
 LICENSE="GPL-3+"
 SLOT="0"
-IUSE="+adblock pdf widevine"
+IUSE="+adblock apparmor pdf widevine"
```

Comment:
Not familiar with apparmor but does this really need a USE? I don't see an issue with just installing these unconditionally (small files). Not like we need to depend on libapparmor here unlike package that have the USE.

Feel free to revbump 2.5.3 to get them installed, being pure python rebuilds are trivial.

Reply:
I just don't want to clutter user's file system with needless files, but personally it's fine either way.

Reply:
Well, it's a policy for small files (e.g. openrc init scripts still get installed on a systemd system), users that really don't want this can always INSTALL_MASK /etc/apparmor.d.

https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0301